crates/sui-indexer-alt-e2e-tests/tests/graphql_available_range_tests.rs

```diff
@@ -216,6 +330,102 @@ async fn cluster_with_pipelines(pipeline: PipelineLayer) -> FullCluster {
     .expect("Failed to create cluster")
 }
 
+fn extract_digests(resp: &Value, path: &[&str], is_event: bool) -> Vec<TransactionDigest> {
+    path.iter()
+        .fold(&resp["data"], |acc, &key| &acc[key])
+        .as_array()
+        .unwrap()
+        .iter()
+        .map(|node| {
+            let digest = if is_event {
+                node["transaction"]["digest"].as_str()
+            } else {
+                node["digest"].as_str()
+            };
+            TransactionDigest::from_str(digest.unwrap()).unwrap()
+        })
+        .collect()
+}
+
+fn assert_tx_digests_eq(expected: &[TransactionDigest], resp: &Value) {
+    assert_eq!(
+        expected,
+        extract_digests(resp, &["transactions", "nodes"], false)
+    );
+}
+
+fn assert_ev_tx_digests_eq(expected: &[TransactionDigest], resp: &Value) {
+    assert_eq!(expected, extract_digests(resp, &["events", "nodes"], true));
+}
+
+fn assert_sequence_numbers_eq(expected_first: u64, expected_last: u64, resp: &Value) {
```

Comment:
Looking at how these work, you could instead choose to create a `sequence_numbers` function that returns a `(u64, u64)` tuple, and then call a regular `assert_eq!` on its output to get the best of both worlds.